static/js/components/CommentList.jsx

```diff
@@ -50,7 +51,10 @@ const CommentList = () => {
     <div className={styles.comments}>
       {items}
       <br />
-      <CommentEntry addComment={addComment} />
+      {
```

Comment:
Curious about logic; would a user not sometimes have comment rights *only on certain sources*?

Reply:
Hmm, good question... We don't currently have that concept, but it's something we should consider adding. Currently the endpoint handler just checks whether the user has the "Comment" ACL: `@permissions(['Comment'])` - I'll open an issue to look into adding per-source (or per-group) permissions